cmd/prestd/main.go

```diff
@@ -1,11 +1,28 @@
 package main
 
 import (
+	"fmt"
+	"plugin"
+
 	"github.com/prest/prest/cmd"
 	"github.com/prest/prest/config"
 )
 
 func main() {
+
+	// TODO: POC load lib (.so)
+	p, err := plugin.Open("./exts/lib/hello.so")
```

Comment:
it needs to check OS before load, because as you can see https://golang.org/pkg/plugin/ plugin is not supported by windows OS.

Reply:
Unfortunately, it is an OS limitation, we will work to load only in UNIX like

If the Go language core supports Windows in the future we will update the documentation and evolve the implementation